cmd/evm/internal/t8ntool/execution.go

```diff
@@ -428,10 +427,16 @@ func (pre *Prestate) Apply(vmConfig vm.Config, chainConfig *params.ChainConfig,
 
 func MakePreState(db ethdb.Database, chainConfig *params.ChainConfig, pre *Prestate, verkle bool) *state.StateDB {
 	// Start with generating the MPT DB, which should be empty if it's post-verkle transition
-	mptSdb := state.NewDatabaseWithConfig(db, &trie.Config{Preimages: true, Verkle: false})
-	statedb, _ := state.New(types.EmptyRootHash, mptSdb, nil)
+	sdb := state.NewDatabaseWithConfig(db, &trie.Config{Preimages: true, Verkle: false})
```

Comment:
shouldn't you set `Verkle` to the value of `verkle` here then?

Reply:
I guess, but interestingly this branch has worked on verkle-genesis with this set to `false`. I'll change it and we can see if causes a problem (which would be surprising).